slinkity 0.5.0 drags in react-dom as a dependency even if you don't use it

I had a project running fine and after running npm update (to grab slinkity 0.5.0) I encountered the following.

npm start

```
package.json
...
"start": "BUILD='development' slinkity --serve --incremental",

```

Yielded the following error
```

...
Pre-bundling dependencies:
  @andystevenson/theme-toggle
  slinkity/client
(this will be run only when your dependencies or config have changed)
✘ [ERROR] Could not resolve "react-dom"

    node_modules/slinkity/client/renderer.js:2:21:
      2 │ import ReactDOM from 'react-dom'
        ╵                      ~~~~~~~~~~~

  You can mark the path "react-dom" as external to exclude it from the bundle, which will remove
  this error.
....

[11ty] Copied 25 files / Wrote 10 files in 0.75 seconds (75.0ms each, v1.0.0)
[11ty] Unhandled rejection in promise: (more in DEBUG output)
[11ty] > Build failed with 1 error:
node_modules/slinkity/client/renderer.js:2:21: ERROR: Could not resolve "react-dom"

`Error` was thrown:
[11ty]     Error: Build failed with 1 error:
    node_modules/slinkity/client/renderer.js:2:21: ERROR: Could not resolve "react-dom"


```

Obviously didn't expect the crash.
I am using slinkity to build a pure eleventy/vite site not using react or any other framework.

Vite now drags in the following dependencies.
```

  slinkity/client
  react
  react/jsx-runtime
  react/jsx-dev-runtime
  (...and 1 more)
```

Which aren't used anywhere in my code.

--- a/packages/slinkity/cli/vite.js
+++ b/packages/slinkity/cli/vite.js
@@ -49,7 +49,7 @@ async function getSharedConfig(eleventyDir) {
         dedupe: ['react', 'react-dom'],
       },
       optimizeDeps: {
-        include: ['react', 'react/jsx-runtime', 'react/jsx-dev-runtime', 'react-dom'],
+        include: ['react', 'react/jsx-runtime', 'react/jsx-dev-runtime', 'react-dom', 'slinkity/client'],
       },
       build: {
         rollupOptions: {
@@ -71,9 +71,7 @@ async function getSharedConfig(eleventyDir) {
       resolve: {
         alias: Object.fromEntries(importAliasesToResolvedPath),
       },
-      optimizeDeps: {
-        include: ['slinkity/client'],
-      },
+      optimizeDeps: {},
     }),
     reactConfig,
   )

--- a/packages/slinkity/cli/vite.js
+++ b/packages/slinkity/cli/vite.js
@@ -49,7 +49,13 @@ async function getSharedConfig(eleventyDir) {
         dedupe: ['react', 'react-dom'],
       },
       optimizeDeps: {
-        include: ['react', 'react/jsx-runtime', 'react/jsx-dev-runtime', 'react-dom', 'slinkity/client'],
+        include: [
+          'react',
+          'react/jsx-runtime',
+          'react/jsx-dev-runtime',
+          'react-dom',
+          'slinkity/client',
+        ],
       },
       build: {
         rollupOptions: {

